packages/canvas/canvas-graphics/test/CanvasGraphicsRenderer.tests.ts

```diff
@@ -4,17 +4,17 @@ import { CanvasRenderer } from '@pixi/canvas-renderer';
 import { CanvasGraphicsRenderer } from '@pixi/canvas-graphics';
 import { expect } from 'chai';
 
-describe('CanvasGraphicsRenderer', function ()
+describe('CanvasGraphicsRenderer', () =>
 {
-    it('should not create pattern for White Texture', function ()
+    it('should not create pattern for White Texture', () =>
     {
         const renderer = new CanvasRenderer({ width: 1, height: 1 });
 
         renderer.plugins.graphics = new CanvasGraphicsRenderer(renderer);
 
         const graphics = new Graphics();
 
-        graphics.beginFill({ texture: Texture.WHITE });
+        graphics.beginTextureFill({ texture: Texture.WHITE });
```

Comment:
Not entirely sure what the intent for this test was. The usage suggests it was meant to be `beginTextureFill`